packages/runtime/src/next/client.ts

```diff
@@ -658,20 +668,33 @@ export class Makeswift {
     {
       siteVersion: siteVersionPromise,
       locale,
-    }: { siteVersion: MakeswiftSiteVersion | Promise<MakeswiftSiteVersion>; locale?: string },
+      allowLocaleFallback = true,
+    }: {
+      siteVersion: MakeswiftSiteVersion | Promise<MakeswiftSiteVersion>
+      locale?: string
+      allowLocaleFallback?: boolean
+    },
   ): Promise<MakeswiftComponentSnapshot> {
     const searchParams = new URLSearchParams()
     if (locale) searchParams.set('locale', locale)
 
     const siteVersion = await siteVersionPromise
-    const response = await this.fetch(
+    const key = deterministicUUID({ id, locale, seed: this.apiKey.split('-').at(0) })
+    const baseLocaleWasRequested = locale == null
```

Comment:
Why not initialize this to `false` and then set it to `true` if the base locale is requested?

Reply:
Never mind, I didn't understand what this meant. I thought this meant that we sent a request to get the base locale as a fallback but this seems to be for whether the caller is requesting a base locale or not